public/index.php

```diff
@@ -3,38 +3,30 @@
 declare(strict_types=1);
 
 use Bolt\Kernel;
-use Symfony\Component\ErrorHandler\Debug;
+
 use Symfony\Component\Dotenv\Dotenv;
+use Symfony\Component\ErrorHandler\Debug;
 use Symfony\Component\HttpFoundation\Request;
 
-require __DIR__.'/../vendor/autoload.php';
-
-// The check is to ensure we don't use .env in production
-if (! isset($_SERVER['APP_ENV'])) {
-    if (! class_exists(Dotenv::class)) {
-        throw new \RuntimeException('APP_ENV environment variable is not defined. You need to define environment variables for configuration or add "symfony/dotenv" as a Composer dependency to load variables from a .env file.');
-    }
-    (new Dotenv())->load(__DIR__.'/../.env');
-}
+require dirname(__DIR__).'/vendor/autoload.php';
 
-$env = $_SERVER['APP_ENV'] ?? 'dev';
-$debug = (bool) ($_SERVER['APP_DEBUG'] ?? ($env !== 'prod'));
+(new Dotenv())->bootEnv(dirname(__DIR__).'/.env');
 
-if ($debug) {
+if ($_SERVER['APP_DEBUG']) {
     umask(0000);
 
     Debug::enable();
 }
 
-if (! empty($_SERVER['TRUSTED_PROXIES'])) {
-    Request::setTrustedProxies(explode(',', $_SERVER['TRUSTED_PROXIES']), Request::HEADER_X_FORWARDED_ALL ^ Request::HEADER_X_FORWARDED_HOST);
+if ($trustedProxies = $_SERVER['TRUSTED_PROXIES'] ?? false) {
+    Request::setTrustedProxies(explode(',', $trustedProxies), Request::HEADER_X_FORWARDED_ALL ^ Request::HEADER_X_FORWARDED_HOST);
```

Comment:
Warning, with Docker if we want everything to work correctly, it has to be like this

`Request::setTrustedProxies(explode(',', $trustedProxies), Request::HEADER_X_FORWARDED_ALL);`

Reply:
This is the same as in https://github.com/bolt/project/blob/master/public/index.php 

Does Docker break on the `^ Request::HEADER_X_FORWARDED_HOST` part?

Reply:
I just corrected that in this [pull request](https://github.com/bolt/project/pull/47/files)

Because by default the project works with Composer and the SQLite database.
With hindsight I think I removed it not exactly for docker but to add a varnish configuration to the project and that's when I had to remove this part.

Since I haven't added Varnish yet, there is obviously no point in removing it.